wpilibj/src/test/java/edu/wpi/first/wpilibj/shuffleboard/ShuffleboardInstanceTest.java

```diff
@@ -130,9 +132,13 @@ void testDuplicateSelectTabs() {
     int listener = 0;
     AtomicInteger counter = new AtomicInteger();
     try {
+      StringSubscriber subscriber =
+          m_ntInstance
+              .getStringTopic("/Shuffleboard/.metadata/Selected")
+              .subscribe("", PubSubOption.keepDuplicates(true));
```

Comment:
The test needs to close the subscriber.  Since there's already a try block here, I would recommend just using try-with-resources.